docs/triage role expectations.md

```diff
@@ -0,0 +1,100 @@
+* `is: enhancement` is to be applied on any issues/PRs that relate to adding, modifying, or removing functionality in 
+core, web, or individual world implementations.
+* `is: refactor/cleanup` is to be applied on any issues/PRs that relate to reorganizing existing code to improve 
+readability or performance without adding, modifying, or removing functionality or fixing known regressions.
+* `is: maintenance` is to be applied on any issues/PRs that don't modify logic, refactor existing code, change features.
+This is typically reserved for pull requests that need to update dependencies or increment version numbers without 
+resolving existing issues.
+* `is: new game` is to be applied to any pull requests that introduce a new game for the first time to the `worlds` 
+directory. 
+    * Issues should not be opened and classified with `is: new game`, and instead should be directed to the 
+    #future-game-design channel in Archipelago for opening suggestions. If they are opened, they should be labeled 
+    with `meta: invalid` and closed.
+    * Pull requests for new games should only have this label, as enhancement, documentation, bug/fix, refactor, and 
+    possibly maintenance is implied.
+
+### Meta
+
+These labels allow additional quick meta information for contributors or reviewers for issues and pull requests. They 
+have specific situations where they should be applied.
+
+* `meta: duplicate` is to be applied to any issues/PRs that are duplicate of another issue/PR that was already opened. 
+    * These should also be immediately closed afterwards after directing to the original request.
+* `meta: invalid` is to be applied to any issues/PRs that do not relate to Archipelago or are inappropriate for 
+discussion on GitHub.
+    * These should also be immediately closed afterwards.
+* `meta: help wanted` is to be applied to any issues/PRs that require additional attention for whatever reason.
+    * These should also include a comment describing what kind of help is requested when the label is added.
+    * Some common reasons include, but are not limited to: Breaking API changes that require developer input/testing or 
+    pull requests with large line changes that need additional reviewers to be reviewed effectively.
+    * This label may require some programming experience and familiarity with Archipelago source to determine if 
+    additional help is warranted.
+* `meta: good first issue` is to be applied to any issues that may be a good starting ground for new contributors to try 
+and tackle.
+    * This label may require some programming experience and familiarity with Archipelago source to determine if an 
+    issue is a "good first issue".
+* `meta: wontfix` is to be applied for any issues/PRs that are opened that will not be actioned because it's out of 
+scope or determined to not be an issue. 
+    * This should be reserved for use by a world's codeowner(s) on its relevant world or by core maintainers.
```

Comment:
```suggestion
* `meta: duplicate` is to be applied to issues/PRs that are duplicate of another issue/PR that was already opened. 
    * These should also be immediately closed afterwards after leaving a comment, directing them to the original request.
* `meta: invalid` is to be applied to issues/PRs that do not relate to Archipelago or are inappropriate for 
discussion on GitHub.
    * These should be immediately closed afterwards.
* `meta: help wanted` is to be applied to issues/PRs that require additional attention for any reason.
    * A comment should be included describing what kind of help is requested when the label is added.
    * Common reasons include, but are not limited to: Breaking API changes that require developer input/testing or 
    pull requests with large line changes that need additional reviewers to be reviewed effectively.
    * This label may require some programming experience and familiarity with Archipelago source to determine if 
    requesting additional help is warranted.
* `meta: good first issue` is to be applied to any issues that may be a good starting ground for new contributors to tackle.
    * This label may require some programming experience and familiarity with Archipelago source to determine if an 
    issue is a "good first issue".
* `meta: wontfix` is to be applied for issues/PRs that will not be actioned upon because they're out of 
scope or determined to not be an issue. 
    * This should be reserved for use by a world's codeowner(s) on their relevant world or by core maintainers.
```